dev/io.openliberty.wsoc.2.2.internal_fat/fat/src/io/openliberty/wsoc22/tests/SendRequestSession.java

```diff
@@ -0,0 +1,143 @@
+
+/*******************************************************************************
+ * Copyright (c) 2022, 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.wsoc22.tests;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.util.logging.Logger;
+
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.fat.util.browser.WebBrowser;
+import com.ibm.ws.fat.util.browser.WebBrowserFactory;
+import com.ibm.ws.fat.util.browser.WebResponse;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+import org.asynchttpclient.ws.*;
+import org.asynchttpclient.*;
+import io.netty.util.concurrent.*;
+/**
+ * WebSocket 2.2 Test
+ */
+@RunWith(FATRunner.class)
+public class SendRequestSession {
```

Comment:
I would comment here 

`Verifies SendResult#getSession works per Spec #185`

`https://github.com/jakartaee/websocket/issues/185` 

May change the name to Spec185GetSessionTest